lib/nexpose/password_policy.rb

```diff
@@ -0,0 +1,69 @@
+module Nexpose
+  require 'json'
```

Comment:
I disagree. It's good practice to require anything used within the same file. We don't do it much in the gem today, but if we moved to it we could start doing things like lazy loading files.

It's a matter of debate, but it's perfectly harmless to leave since it will only do the work once.